### PR TITLE
!!! TASK: Raise minimum PHP version to 7.3

### PR DESCRIPTION
**Pull request by albe**
See https://github.com/neos/flow-development-collection/issues/2236

### CODE REVIEW

**Comment by albe** [comment]
Only in main composer and this one package for Neos. @kdambekalns should this maybe be changed or is the transitive dependency on Flow enough?

**Comment by kdambekalns** [comment]
> is the transitive dependency on Flow enough?

Hm, seems a bit weak. It would be better to require a PHP version in all packages, I think. They are "standalone" after being split, after all, and e.g. https://packagist.org/packages/neos/neos says nothing about the PHP version…

WDYT @kitsunet, @robertlemke, @bwaidelich?

**Review by kdambekalns — COMMENTED** [review]
Cleaning up the collection manifest is useless, btw, since it is (sopposed to be) autogenerated using the `ComposerManifestMerger.php` script…

**Comment by albe** [comment]
The other packages are only Fusion/Settings, so no PHP involved directly

**Comment by albe** [comment]
Tests fail due to https://github.com/neos/flow-development-collection/pull/2240

**Comment by jonnitto** [comment]
What's about PHP Version 8? In Flow, we support also Version 8…

**Comment by albe** [comment]
> What's about PHP Version 8? In Flow, we support also Version 8…

Well, we do on the paper (=composer.json), but doctrine does not yet support PHP 8, so until they do, we can't even run tests (see https://github.com/neos/flow-development-collection/issues/2233). I think adding support for PHP 8 can be done pretty much at any point, once we are sure about it and know what (if anything) needs to be changed.

**Comment by albe** [comment]
Everything green now. Good to go IMO

File: Neos.Neos/Tests/Unit/Fusion/PluginImplementationTest.php [hunk 1]
```diff
@@ -135,7 +135,7 @@ public function evaluateSetHeaderIntoParent(string $message, array $input, array
         $this->pluginImplementation->evaluate();
 
         foreach ($expected as $expectedKey => $expectedValue) {
-            self::assertEquals($expectedValue, (string)$parentResponse->getComponentParameters()[SetHeaderComponent::class][$expectedKey], $message);
+            self::assertEquals($expectedValue, (string)$parentResponse->getHttpHeader($expectedKey), $message);
```
Comment (albe) [inline Neos.Neos/Tests/Unit/Fusion/PluginImplementationTest.php:138]: See also #3175 but need this here to pass tests.

**Review by jonnitto — APPROVED** [review]

**Review by kdambekalns — COMMENTED** [review]
Post-merge 👍

**Comment by bwaidelich** [comment]
We need to update https://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartII/Requirements.html as well

**Pull request merged**